WordPress/Tests/WP/UseCapabilitiesNotRolesUnitTest.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+
+
+add_dashboard_page( 'page_title', 'menu_title', 'super_admin', 'menu_slug', 'function'); // Error.
+add_posts_page( 'page_title', 'menu_title', 'administrator', 'menu_slug', 'function' ); // Error.
+add_media_page( 'page_title', 'menu_title', 'editor', 'menu_slug', 'function' ); // Error.
+add_pages_page( 'page_title', 'menu_title', 'author', 'menu_slug', 'function' ); // Error.
+add_comments_page( 'page_title', 'menu_title', 'contributor', 'menu_slug', 'function' ); // Error.
+add_theme_page( 'page_title', $menu_title , 'subscriber', 'menu_slug', 'function' );  // Error.
+add_plugins_page( 'page_title', 'menu_title', 'super_admin', 'menu_slug', 'function' );  // Error.
+add_users_page( 'page_title', 'menu_title', 'administrator', 'menu_slug', 'function' );  // Error.
+add_management_page( 'page_title', 'menu_title', 'editor', 'menu_slug', 'function' );  // Error.
+add_options_page( 'page_title', 'menu_title', 'contributor', 'menu_slug', 'function' );  // Error.
+// @codingStandardsChangeSetting WordPress.WP.UseCapabilitiesNotRoles checkOnlyKnownCaps false
```

Comment:
Please add a blank line above this to make it easier to see that a property is being changed and something else is now being tested.

Reply:
Also: please _reset_ the property at the end of the block of tests so additional unit tests which may be added later can expect the default properties to be in effect.